control/plugin/plugin.go

```diff
@@ -262,7 +263,7 @@ type Response struct {
 // requestString - plugins arguments (marshaled json of control/plugin Arg struct)
 // returns an error and exitCode (exitCode from SessionState initilization or plugin termination code)
 func Start(m *PluginMeta, c Plugin, requestString string) (error, int) {
-	log.SetFlags(0)
+	log.SetFormatter(&log.TextFormatter{DisableTimestamp: true})
```

Comment:
I think this is removing the timestamp from the plugin logs since a timestamp will be added by snapd's logs.